python/GafferSceneTest/IECoreScenePreviewTest/CapturingRendererTest.py

```diff
@@ -165,5 +168,390 @@ def testDeformingObject( self ) :
 		self.assertEqual( c.capturedSamples(), [ sphere1, sphere2 ] )
 		self.assertEqual( c.capturedSampleTimes(), [ 1, 2 ] )
 
+	class TestProcedural( GafferScene.Private.IECoreScenePreview.Procedural ) :
+
+		def __init__( self, n ) :
+
+			GafferScene.Private.IECoreScenePreview.Procedural.__init__( self )
+			self.n = n
```

Comment:
As far as I can tell, this is unused, and the constructor argument is unnecessary?